B827EBFFFE7549C1.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFE7549C1",
+    "servers": [
+      {
+        "server_address": "ttn-handler-us-west",
```

Comment:
The address to the server is wrong, you need to enter a router address, not a handler one (in any case, that's not a valid handler address either). Check the official routers addresses here: 
https://www.thethingsnetwork.org/wiki/Backend/Connect/Gateway